### ISSUE TITLE
cmake CANNOT LINK EXECUTABLE "cmake": cannot locate symbol "_ZN4Json5ValueaSES0_"

### ISSUE THREAD

**Issue by JieJiSS** [termux/termux-packages#4944]
**Problem description**

Exactly the same issue as #4026, except that I encountered this error on `cmake=3.15.5-1`, which, according to [this comment](https://github.com/termux/termux-packages/issues/4026#issuecomment-509013508), is newer than `3.14.5-2`, so it should have fixed the error. Do you have any idea about this?

My Android structure is `aarch64`.

**Steps to reproduce**

```shell
$ apt install cmake   # or pkg install cmake
$ cmake
cmake CANNOT LINK EXECUTABLE "cmake": cannot locate symbol "_ZN4Json5ValueaSES0_"
```

**Expected behavior**

`cmake` should work properly.

**Additional information**

Not provided.

**Comment by xeffyr** [comment]
I can't reproduce this on my aarch64 device.

You also didn't provided the output of `termux-info`, so I'm closing issue as "unreproducible". Make sure all your packages are up-to-date.

**Comment by JieJiSS** [comment]
In case that anyone may want to know: performing a full reinstallation of termux and reinstalling the same version of `cmake`  fixed the issue. (I had to do this because `apt upgrade` broke the environment.)

My failure to provide `termux-info` output was a consequence of manually compiling `cmake` from source code after I had realized thay my `cmake` was broken. It took me several hours, and when I'm creating this issue, it was still compiling `cmake` thus no place for me to run commands.

Still, I have no clue about what was wrong. But anyway, the issue was fixed, and thanks for your help.
